apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -68,6 +68,12 @@ function WelcomeTourFrame() {
 	const [ justMaximized, setJustMaximized ] = useState( false );
 
 	const { setShowWelcomeGuide } = useDispatch( 'automattic/wpcom-welcome-guide' );
+	const { setOpenState } = useDispatch( 'automattic/starter-page-layouts' );
+
+	const { shouldOpenPatternsPanel, isManuallyOpened } = useSelect( ( select ) => ( {
+		shouldOpenPatternsPanel: select( 'automattic/wpcom-welcome-guide' ).shouldOpenPatternsPanel(),
```

Comment:
Thinking this sort of thing can be renamed now?

```suggestion
		shouldOpenPagePatternsModal: select( 'automattic/wpcom-welcome-guide' ).shouldOpenPagePatternsModal(),
```